### PR TITLE
fix: button sizing changes due to active border

### PR DESCRIPTION
**Pull request by nickrm97**
# Objective
`FilterMenuButton` currently has a `2px` border on active state, which causes the overall button height and width to grow on click. This behaviour is quite janky, and to solve this, a padding matching the border sizing is added when the button is **not** active.

# Motivation and Context
As part of #team_perform_perfx's changes to Admin Goals Reporting page

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

File: draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss [hunk 1]
```diff
@@ -119,6 +119,11 @@
   letter-spacing: $kz-var-typography-button-secondary-letter-spacing;
 }
 
+// Account for border width affecting overall button size
+.buttonNotActive {
```
Comment (lisatassone) [inline draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss:123]: Could we perhaps always have a border? We can make it transparent when it's not active and add the active color when it is?

**Review by lisatassone — COMMENTED** [review]
I'd try my border suggestion.

File: draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss (hunk 1 above)
Comment (nickrm97) [reply draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss:123]: Yeah that's a cleaner solution, I've pushed up that change now

**Review by lisatassone — APPROVED** [review]
🎉

**Review by martindavid — APPROVED** [review]

**Pull request merged**